spec/factories/users.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  sequence(:email) { |n| "test#{n}@test.com" }
```

Comment:
This corrupts the global scope of FactoryBot. You should consider adding a setup file or a helpers file or something so that this is not hidden here